### PR TITLE
BaSP-TG-71: Employees Form

### PR DESCRIPTION
**Pull request by HernanCazeneuve**
Captura de pantalla 2022-06-07 191556
Captura de pantalla 2022-06-07 191649

### CODE REVIEW

File: src/Components/Employee/index.js [hunk 1]
```diff
@@ -80,44 +88,89 @@ const EmployeeForm = () => {
...
+  const history = useHistory();
+  const routeChange = () => {
+    let path = `/employees`;
+    history.push(path);
+  };
+
   return (
-    <section className={styles.containerSec}>
+    <div className={styles.containerSec}>
       <h3>Add admin</h3>
```
Comment (CarVigil) [inline src/Components/Employee/index.js:107]: I think the title must be Add Employee

File: src/Components/Employee/employee.module.css [hunk 2]
```diff
@@ -41,7 +43,7 @@ a {
 }
 
 .button {
```
Comment (CarVigil) [inline src/Components/Employee/employee.module.css:45]: This style isn't used. It could be deleted.

**Review by CarVigil — CHANGES_REQUESTED** [review]

File: src/Components/Employee/employee.module.css [hunk 3]
```diff
@@ -9,58 +9,28 @@
...
 .buttonContainer {
   width: 100%;
   display: flex;
   justify-content: center;
   align-items: center;
   padding: 0 25px;
+  margin: 10px;
 }
 
 a {
```
Comment (Gearyandres94) [inline src/Components/Employee/employee.module.css:21]: Do not use element selectors.

File: src/Components/Employee/index.js [hunk 4]
```diff
@@ -80,44 +88,89 @@ const EmployeeForm = () => {
...
+            type="date"
+            value={dateOfBirth}
+            placeholder="Enter your Birth date"
+            onChange={setDateOfBirth}
+          />
+          <Input
+            name="Dni"
+            type="number"
+            value={dni}
+            placeholder="Enter your IDN"
```
Comment (Gearyandres94) [inline src/Components/Employee/index.js:135]: This should be DNI

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
The error that you recieves from the validations of the backend should be a modal. If not the design is broken.
Capture

**Review by Gearyandres94 — APPROVED** [review]
Great Job!!

**Review by CarVigil — APPROVED** [review]

**Review by Facund0Fernandez — DISMISSED** [review]

**Review by tobiaspujol — APPROVED** [review]
PR ticket comment corrected!

**Review by Gearyandres94 — DISMISSED** [review]
Good Job!!

**Review by GuidoCerioni — DISMISSED** [review]
Nice job

**Review by Gearyandres94 — APPROVED** [review]

**Review by CarVigil — APPROVED** [review]
Perfect! 👌

**Review by Gearyandres94 — APPROVED** [review]

**Review by Facund0Fernandez — APPROVED** [review]

**Pull request merged**